subprojects/internal-testing/src/test/groovy/org/gradle/util/SetSystemPropertiesTest.groovy

```diff
@@ -1,78 +0,0 @@
-/*
```

Comment:
do we need this to test that system properties are restored with this rule?

Reply:
Probably should - I've added tests for it.